src/pages/ProductDetail/component/ProductImformation.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ProductImformation.scss';
+
+const ProductImformation = () => {
+  return (
+    <div className="product-information">
+      <div className="description-button-box">
+        <button className="description-button">상품설명</button>
+        <button className="description-button">상세정보</button>
+        <button className="description-button">후기(2,312)</button>
```

Comment:
```suggestion
        <button className="description-button description">상품설명</button>
        <button className="description-button detail">상세정보</button>
        <button className="description-button review">후기(2,312)</button>
```
추가하면 구체적으로 알기 쉬울것 같아요.